Tests/Tests/Rows/BindingTests.swift

```diff
@@ -17,6 +17,8 @@ class BindingTestCase: BaseTestCase {
     // MARK: - Helper Types
 
     private struct Person: CodableBinding, Equatable {
+        typealias BindingType = Data
```

Comment:
Here's an example @AtomicCat of the change clients would be required to make on Swift 4.1.